recipe/meta.yaml

```diff
@@ -25,6 +25,8 @@ requirements:
     - python 2.7.*  # [not win]
   host:
     - vs2015_runtime  # [win]
+    - git  # [linux]
+    - make  # [linux]
```

Comment:
These should be in `build`

Reply:
@isuruf I thought so too but my my multiarch builds were failing when I had these packages in the build step.  From what I could tell the multiarch Docker images don't contain `make` while the x86 image does.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=31241

Reply:
I mean you have these in `host` section. They should be in `build` section

Reply:
The link I posted to is what happens when those are defined in the build section.  Maybe there was something I missed, I will try changing back and see.

Reply:
Update seems to have worked.